site/includes/placeholdercontent.hbs

```diff
@@ -103,4 +103,5 @@
 </form>
 <blockquote>
 	<p>"<code>blockquote</code>"</p>
+	<footer><cite><code>cite</code></cite></footer>
```

Comment:
What about changing the text "cite" by "footer>cite"? I think that change will be more obvious that the "cite" need to be in a "footer" element

Reply:
IMO ``footer>cite`` would look kind of strange. I'd prefer not to place anything resembling a CSS selector into the placeholder content.

I could compromise by placing ``footer`` ``cite`` side-by-side. That would at least make it more obvious at first glance that a ``footer`` element is involved in creating that layout. Plus it's technically ok for the ``footer`` element to have content in it separately from ``cite`` in that context.